### ISSUE TITLE
Exception information missing

### ISSUE THREAD

**Issue by geewiz** [palkan/action_policy#13]
### Tell us about your environment

**Ruby Version:** 2.4

**Framework Version (Rails, whatever):** 5.0

**Action Policy Version:** 0.1.4

### What did you do?

Handling an authorization exception as described in the documentation

### What did you expect to happen?

Failure reason messages to be returned

### What actually happened?

There's no `reasons.messages`. The string `messages` appears exactly twice in the source code — in a class file comment and the doc file that is generated from it.

**Comment by palkan** [comment]
Oops, sorry, this feature hasn't been pushed to GitHub yet.

I've decided to reconsider it a little bit and make look more like [`ActiveModel::Errors`](http://api.rubyonrails.org/classes/ActiveModel/Errors.html):

```ruby
reasons.details #=> { user: [:create?, :update?] }

# as a part of i18n integration
reasons.messages #=> { user: ["can not create", "can not update"] }
reasons.full_messages #=> ["can not create user", "can not update user"]
```

I've updated the docs.

**Comment by palkan** [comment]
`reasons.details` has been released in 0.2.0. See the updated [docs](https://actionpolicy.evilmartians.io/#/reasons).
